File: LV 2/피보나치 수.py
def solution(n):
    answer = 0
    a = 0
    b = 1
    c = 1
    if n == 1:
        return 1
    
    for i in range(2, n+1):
        c = a + b
        a = b
        b = c
    
    answer = c % 1234567
    return answer

File: LV 2/피보나치 수.java
class Solution {
    
    public int fibo(int n) {
        int a = 0;
        int b = 1;
        for (int i = 2; i <= n; i++) {
            int c = (a + b) % 1234567;
            a = b;
            b = c;
        }
        
        return b % 1234567;
    }
    
    public int solution(int n) {
        return fibo(n);
    }
}
